src/background.js

```diff
@@ -33,6 +34,7 @@ function updateButtonForTab(tabId, changeInfo) {
 
 function init() {
   log('init()');
+  Metrics.init(window.BroadcastChannel, browser.tabs);
```

Comment:
Basically, it's dependency injection, so that the tests don't have to rely on a `window` global to pass in a `BroadcastChannel` class because it's not an API based on an importable module